ada_2_fhir-r4/zibs2020/payload/0.7-dev/nl-core-CareAgreement.xsl

```diff
@@ -0,0 +1,147 @@
+                                <xsl:with-param name="profile" select="'nl-core-HealthProfessional-PractitionerRole'"/>
+                            </xsl:call-template>
+                        </xsl:if>
+                        <xsl:if test="contactpersoon">
+                            <xsl:call-template name="makeReference">
+                                <xsl:with-param name="in" select="$careCoordinator"/>
+                                <xsl:with-param name="profile" select="'nl-core-ContactPerson'"/>
+                            </xsl:call-template>
+                        </xsl:if>
+                        <xsl:if test="patient">
+                            <xsl:call-template name="makeReference">
+                                <xsl:with-param name="in" select="$careCoordinator"/>
+                                <xsl:with-param name="profile" select="'nl-core-Patient'"/>
+                            </xsl:call-template>
+                        </xsl:if>
+                    </author>
+                </xsl:for-each>
+                <xsl:for-each select="$agreementParties">
+                    <contributor>
+                        <xsl:call-template name="makeReference">
+                            <xsl:with-param name="in" select="$agreementParties"/>
+                            <xsl:with-param name="profile" select="'nl-core-HealthProfessional-PractitionerRole'"/>
+                        </xsl:call-template>
+                    </contributor>
+                </xsl:for-each>
+                <xsl:call-template name="makeReference">
+                    <xsl:with-param name="in" select="$problem"/>
+                    <xsl:with-param name="profile" select="'nl-core-Problem'"/>
+                    <xsl:with-param name="wrapIn" select="'addresses'"/>
+                </xsl:call-template>
+                <activity>
+                    <detail>
+                        <xsl:for-each select="activiteit">
+                            <code>
+                                <text value="{@value}"/>
+                            </code>
+                            </xsl:for-each>
+                            <!-- Follows implicit zib mapping to StartDate and EndDate as described in the profile, 'in-progress' == 'active'. -->
+                            <status>
+                                <xsl:attribute name="value" select="'in-progress'"/>
+                            </status>
+                        <xsl:call-template name="makeReference">
+                            <xsl:with-param name="in" select="$performer"/>
+                            <xsl:with-param name="profile" select="'nl-core-HealthProfessional-PractitionerRole'"/>
+                            <xsl:with-param name="wrapIn" select="'performer'"/>
+                        </xsl:call-template>
+                        <xsl:for-each select="toelichting">
+                            <description value="{normalize-space(@value)}"/>
```

Comment:
Maybe use `string-to-string` here as well (I think we need to make this consistent over all stylesheets, actually).